UnrealPlugin/ThirdPersonMPCloudDeploy.md

```diff
@@ -15,28 +15,81 @@ This guide explains how to deploy your game to PlayFab Multiplayer Server (MPS)
 * [Completed Unreal Project](ThirdPersonMPSetup.md) with [PlayFab Unreal GSDK](ThirdPersonMPGSDKSetup.md) installed and configured
 * [Release Server](ThirdPersonMPBuild.md) configuration of your project built from Visual Studio or Development Editor
 * [Client](ThirdPersonMPBuild.md) configuration of your project built from Visual Studio or Development Editor
+* MPS is [enabled for your title](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/enable-playfab-multiplayer-servers), and billing is set up
 
 ## Instructions
 
 ### Packaging the game server
```

Comment:
Nice job of making a list of all the steps. Keeping it simple. Awesome!

Reply:
Will do in near future. Thanks.